libraries/integration/Microsoft.Bot.Builder.Integration.Runtime/Settings/BlobsStorageSettings.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.Integration.Runtime.Settings
```

Comment:
why do we need all these settings classes?

Reply:
To have strongly typed versions of all the knobs that composer exposes.

Reply:
we can evaluate in the next milestone - I think we can get rid of more code

Reply:
Sure. It would be nice to re-think the static service collection extensions along with the moving of these settings to the Options patterns possibly. But needs some group discussion.